sources/mysql/data.go

```diff
@@ -137,9 +137,22 @@ func convScalar(conv *internal.Conv, spannerType ddl.Type, srcTypeName string, T
 	}
 }
 
-func convBool(conv *internal.Conv, val string) (bool, error) {
+func convBool(conv *internal.Conv, spannerType ddl.Type, srcTypeName string, val string) (bool, error) {
 	b, err := strconv.ParseBool(val)
 	if err != nil {
+
+		if srcTypeName == "bit" {
+			// To handle scenarios where bit is used to store boolean
+			// zero treated as false
+			// one treated as true
+			switch val {
+			case "\x00":
```

Comment:
Will this byte string be guaranteed to be 2 digit - either "\x00" or "\x01"? Can source not be "\x1" or "\x001" etc?

Should we instead parse to number if "starts with \x" and compare to 1 or 0?

Reply:
This is how 0 and 1 are represented in bytes in go

Reply:
I see, bit(1) data is always read as these 2 string we should be good